README.md

```diff
@@ -74,7 +74,7 @@ ChuckNorris::actUnless($isChuckNorrisPuny, $data);
 
 The `$data` is passed into the action's constructor. You can pass as many arguments as is needed in your use case.
 
-The second two methods, `actWhen` and `actUnless` require a condition as the first variable. These work like other Laravel methods such as `throw_if()` and `throw_unless()`. And once again, you can pass and many arguments as is needed for your action after the condition.
+The second two methods, `actWhen` and `actUnless` require a condition as the first variable. These work like other Laravel methods such as `throw_if()` and `throw_unless()`. Finally, you can pass as many arguments as needed for your action after the condition.
```

Comment:
Not certain about this just being me, but I often just read what I need so if there's no further explanation or big description, I wouldn't use "again"